packages/grafana-ui/src/components/Cascader/Cascader.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+// @ts-ignore
+import RCCascader from 'rc-cascader';
+import { BuildInPlacements as BuiltInPlacements } from 'rc-trigger';
+
+export interface CascaderOption {
+  label: string;
+  value: string;
+
+  children?: CascaderOption[];
+  disabled?: boolean;
+}
+
+export interface CascaderProps {
```

Comment:
Would be great to limit this API only to the props that we are using. Don't see much value in wrapping a component and exposing its entire API. WDYT?

Reply:
Sure, makes sense to me

Reply:
yea, agree. makes upgrades easier and less risk of breaking plugins that use this :)